src/apps/Settings/DaoSettings.js

```diff
@@ -58,7 +58,7 @@ class DaoSettings extends React.Component {
     const financeApp = apps.find(({ name }) => name === 'Finance')
     const checksummedDaoAddr =
       daoAddress.address && toChecksumAddress(daoAddress.address)
-    const webApps = apps.filter(app => app.hasWebApp)
+    const apmApps = apps.filter(app => app.isOnAPM)
```

Comment:
Could we avoid doing this and just use `!isAragonOsInternalApp` instead?

Reply:
Also wondering if we should have a different section for "non-visible" installed apps?

Reply:
Maybe a tag or a label next to them? Or something in the title, e.g. “Vault (contract-only)”

Reply:
Good idea @sohkai!

Reply:
Adding the contract-only label in #610